doc/manual/src/configuartion.md

```diff
@@ -0,0 +1,130 @@
+Configuration
+=============
+
+This chapter is a collection of configuration snippets for different
+scenarios.
+
+Including files
+---------------
+
+`hydra.conf` supports Apache-style includes. This is **IMPORTANT**
+because that is how you keep your **secrets** out of the **Nix store**.
+Hopefully this got your attention 😌
+
+This:
+```
+<github_authorization>
+NixOS = Bearer gha-secret😱secret😱secret😱
+</github_authorization>
+```
+should **NOT** be in `hydra.conf`.
+
+`hydra.conf` is rendered in the Nix store and is therefore world-readable.
+
+Instead, the above should be written to a file outside the Nix store by
+other means (manually, using Nixops' secrets feature, etc) and included
+like so:
+```
+Include github_authorizations.conf
```

Comment:
```suggestion
Include /run/keys/hydra_github_authorizations.conf
```

Reply:
Oh I haven't been able to use absolute paths. It tries to append them to `/var/lib/hydra`. That's why I mentioned symlinks as a workaround. I haven't checked relative paths tbh. I'll expand the example to demonstrate that.

Regarding the config refactor, from this side of the night it looks to me like `getHydraConfig` doesn't really belong (at least semantically) in `Helpers::Nix`. How about splitting that to a `Helpers::Config` and expose a sub that just returns the `Config::General` options? That would make it more akin to a semi-real refactor. Though perhaps it increases the risk of breaking something so you tell me 😌

Reply:
Oh dear, oops, I guess I missed the part about symlinks. I'm glad I waited for your review!

I would be comfortable with you doing that refactor for sure. Would you like this PR to merge first? I'd say this is a notable improvement in docs and behavior already, and I don't want you to feel like this is becoming annoying gatekeeping.